libr/bin/format/elf/elf.c

```diff
@@ -1731,6 +1731,11 @@ ut64 Elf_(r_bin_elf_get_main_offset)(ELFOBJ *bin) {
 		*/
 		if (*addr > text && *addr < (text_end)) {
```

Comment:
If this check is wrong, it should be removed

Reply:
yeah i agree. except that i don't know the history of this code, so maybe it's correct for some binaries.

Reply:
👍